eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/BasicTypeSpecClientOptions.cs

```diff
@@ -6,12 +6,12 @@
 #nullable disable
 
 using System;
-using System.ClientModel.Primitives;
+using Azure.Core;
 
 namespace BasicTypeSpec
 {
     /// <summary> Client options for <see cref="BasicTypeSpecClient"/>. </summary>
-    public partial class BasicTypeSpecClientOptions : ClientPipelineOptions
+    public partial class BasicTypeSpecClientOptions : ClientOptions
```

Comment:
These will be more different going forward ... just a heads up that more will need to differ here.